flutter/example/pubspec_overrides.yaml

```diff
@@ -15,3 +15,10 @@ dependency_overrides:
     path: ../../hive
   sentry_drift:
     path: ../../drift
+  sentry_isar:
+    path: ../../isar
+  isar_generator: # https://github.com/isar/isar/issues/686#issuecomment-1473668196
```

Comment:
@buenaflor The issue with the specific comment is already linked here. Should we add it elsewhere?